tomviz/DataSource.cxx

```diff
@@ -770,7 +770,6 @@ void DataSource::setData(vtkDataObject* newData)
   auto tp = producer();
   Q_ASSERT(tp);
   tp->SetOutput(newData);
-  newData->FastDelete();
```

Comment:
Can you test this with generated data?  I remember having to tweak some generated data stuff related to this FastDelete call.

Reply:
Worked fine with random particles here

Reply:
I see, you fixed the other side of it too.  +1